tests/shared/hardware_lfs.py

```diff
@@ -8,9 +8,9 @@
     overlap,
 )
 
-ABSTAIN = 0
-FALSE = 1
-TRUE = 2
+ABSTAIN = -1
```

Comment:
How about moving these constants to hardware_utils and import from there?

Reply:
Thanks for your suggestion. Done at 47d94ed0d377d0caf83c87b9ae901532431cb71a.